Introduces UnsupportedPropertiesContext, Utill and use it in Symbolizer Editors (#1844)
geostyler/geostyler: src/context/UnsupportedPropertiesContext/UnsupportedPropertiesContext.example.md

<!--
 * Released under the BSD 2-Clause License
 *

>  * Copyright © 2018-present, terrestris GmbH & Co. KG and GeoStyler contributors

Reviewer: ```suggestion
 * Copyright © 2022-present, terrestris GmbH & Co. KG and GeoStyler contributors
```

 *
 * * Redistributions of source code must retain the above copyright notice,
 *   this list of conditions and the following disclaimer.
 *
 * * Redistributions in binary form must reproduce the above copyright notice,
 *   this list of conditions and the following disclaimer in the documentation
 *   and/or other materials provided with the distribution.
 *
 * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
 * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
 * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
 * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
 * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
 * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
 * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 * POSSIBILITY OF SUCH DAMAGE.
 *
-->

> The `UnsupportedPropertiesContext` deploys unsuported properties to underlying components.

Reviewer: ```suggestion
The `UnsupportedPropertiesContext` deploys unsupported properties to underlying components.
```

> The components themeselve will disable field that are marked as not supported and add notes

Reviewer: ```suggestion
The components themeselves will disable fields that are marked as not supported and add notes
```

> for fields that are only partialy supported.

Reviewer: ```suggestion
for fields that are only partially supported.
```

#### Example

Provide some default values.

```jsx
import * as React from 'react';
import { UnsupportedPropertiesContext, FillEditor } from 'geostyler';
import OlStyleParser from 'geostyler-openlayers-parser';
import SldStyleParser from 'geostyler-sld-parser';
import { Checkbox, Switch, InputNumber } from 'antd';

> class UnsupportedPropertiesContextExample extends React.Component {

Reviewer: Out of curiosity: Is there a reason you are using class components instead of function components?

Author: When writing the Example i thought that only class components are supported as example. :/
